Leanplum-SDK/Classes/Managers/Networking/LPRequestManager.m

```diff
@@ -389,7 +393,18 @@ - (void)sendRequests:(BOOL)async
     }
 }
 
--(void)uiDidTimeout {
+- (void)sendAllCounts {
+    NSDictionary *counts = [[LPCountManager sharedManager] getAndClearCounts];
+    for (id key in counts) { // iterate over counts, creating one request per counter
+        NSMutableDictionary *params = [[NSMutableDictionary alloc] init];
+        params[@"TYPE"] = @"SDK_COUNT";
```

Comment:
we wanna avoid using hard coded strings here. Search for LP_PARAM_TYPE and use that instead. If any of these don't exist, let's create them